BUILD.bazel

```diff
@@ -129,7 +129,7 @@ genrule(
         "aten/src/ATen/RegisterCPU.cpp",
         "aten/src/ATen/RegisterMkldnnCPU.cpp",
         "aten/src/ATen/RegisterQuantizedCPU.cpp",
-        "aten/src/ATen/RegisterSparseCPU.cpp",
+        "aten/src/ATen/RegisterSparseCOO_CPU.cpp",
```

Comment:
this underscore's gonna keep me up at night 🤣

Reply:
I've decided to not do anything to COO in this PR and have removed this one.